app/src/app/components/contents/packets/PacketParameters.tsx

```diff
@@ -0,0 +1,25 @@
+import { Separator } from "../../Base/Separator";
+
+interface PacketParametersProps {
+  parameters: Record<string, string>;
+}
+export const PacketParameters = ({ parameters }: PacketParametersProps) => {
+  return (
+    <div>
+      <h4 className="scroll-m-20 text-xl font-semibold tracking-tight">Parameters</h4>
+      <div className="flex flex-wrap gap-1">
+        {Object.keys(parameters)?.length === 0 ? (
+          <div className="italic text-sm">None</div>
+        ) : (
+          Object.entries(parameters).map(([key, val]) => (
+            <div key={key} className="border p-1 rounded-md flex space-x-1 text-sm">
+              <div>{key}</div>
+              <Separator orientation="vertical" />
```

Comment:
I guess this should now be consistent with the table view, which shows colon now rather than vertical separator